apps/client/src/pages/LocationPage.js

```diff
@@ -67,9 +68,9 @@ const LocationPage = ({ topic }) => {
         <LocationFinder
           setAddress={setAddress}
           setErrorMessage={setErrorMessage}
-          postalCode={context.address?.postalCode}
-          houseNumberFull={context.address?.houseNumberFull}
-          houseNumber={context.address?.houseNumberFull}
+          postalCode={context.autofillData.address?.postalCode}
```

Comment:
Maybe destructure this?

Reply:
Don't waste time on this destructure, the whole thing is going to be changed with session data instead of context data:
```
postalCode={sessionAddress?.postalCode}
houseNumberFull={sessionAddress?.houseNumberFull}
houseNumber={sessionAddress?.houseNumberFull}
```